client/src/App.js

```diff
@@ -65,19 +92,22 @@ function App() {
     return (
         <>
              {
-                sections.length !== 0 ?
+                currentSections.length !== 0 ?
```

Comment:
this statement is not required now?

Reply:
i think it is required to not get error due to null values anywhere